database/index.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require('mongoose');
+
+mongoose.Promise = global.Promise;
+mongoose.connect('mongodb://localhost/image-slider');
+
+let imagesSchema = mongoose.Schema({
+  image_id { type: Number, required: true },
+  url { type: String, required: true }
+
```

Comment:
I'm not entirely sure if this is a lapse in my understanding of mongoose.Schema (in which case you can simply ignore this) but I'm not sure where the variable declaration for imagesScheme ends. It looks like the definition ends at the end of the file but it also gets referenced on line 10, I'm not sure if this causes any potential problems but I think it might be easier to read if the variable wasn't referenced within its declaration.